### PR TITLE
TDP-3842: No secrets in callbackUrls

### PR DESCRIPTION
**Pull request by vsmith-bw**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

## Brief Summary of changes
Added a reminder to use proper auth fields, not to dump secrets into the callbackUrl

### CODE REVIEW

**Review by nashley — COMMENTED** [review]
Do we already have similar language for `tag`?
Could maybe indicate that the URLs are logged on our end to be more clear about what potentially-risky stuff we do with them. I'm mostly ambivalent on that though.

**Comment by vsmith-bw** [comment]
> Do we already have similar language for `tag`?
> Could maybe indicate that the URLs are logged on our end to be more clear about what potentially-risky stuff we do with them. I'm mostly ambivalent on that though.

Jacob said he'd rather not be specific about what we log, so I didn't introduce that language. It looks like we don't have this memo on tags..... Want me to slap that onto the page here as well?

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3842.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3842.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3842.s3-website-us-east-1.amazonaws.com/

**Comment by jmulford-bw** [comment]
Apologies for the bucket weirdness, this link should work

https://bw-dev-tdp-3842.s3.amazonaws.com/index.html

**Comment by marcelohossomi** [comment]
Worth putting this in a highlight box like here? Not sure it would work/look good inside a table though.
https://bw-dev-tdp-3842.s3.amazonaws.com/messaging/callbacks/msgDelivered.html

File: voice/methods/calls/postCalls.md [hunk 1]
```diff
@@ -11,9 +11,11 @@ Creates a new outbound phone call.
 
 Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Credentials. Read more about how Bandwidth secures endpoints in the [Security & Credentials](../../../guides/accountCredentials.md) document.
 
+You should not include sensitive or personally-identifiable information in any tag or url field! Always use the proper username and password fields for authorization.
+
 ---
 
-### Supported Parameters
+### Supported Parameters
```
Comment (n-miles) [inline voice/methods/calls/postCalls.md]: Remove

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3842.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3842.s3-website-us-east-1.amazonaws.com/

**Comment by vsmith-bw** [comment]
> Worth putting this in a highlight box like here? Not sure it would work/look good inside a table though.
> https://bw-dev-tdp-3842.s3.amazonaws.com/messaging/callbacks/msgDelivered.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3842.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3842.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3842.s3-website-us-east-1.amazonaws.com/

File: voice/methods/calls/postCalls.md [hunk 2]
```diff
@@ -11,6 +11,13 @@ Creates a new outbound phone call.
 
 Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Credentials. Read more about how Bandwidth secures endpoints in the [Security & Credentials](../../../guides/accountCredentials.md) document.
 
+
+<aside class="alert general">
+<p>IMPORTANT NOTE ABOUT AUTHORIZATION!</p>
+You should not include sensitive or personally-identifiable information in any tag or url field! Always use the proper username and password fields for authorization.
```
Comment (marcelohossomi) [inline voice/methods/calls/postCalls.md]:
```suggestion
You should not include sensitive or personally-identifiable information in any tag or URL field! Always use the proper username and password fields for authorization.
```

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3842.s3-website-us-east-1.amazonaws.com/

**Review by marcelohossomi — APPROVED** [review]

**Pull request merged**